jetty-http2/http2-server/pom.xml

```diff
@@ -17,11 +17,15 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>com.github.madgnome</groupId>
+        <groupId>org.mortbay.jetty</groupId>
         <artifactId>h2spec-maven-plugin</artifactId>
         <configuration>
           <mainClass>org.eclipse.jetty.http2.server.H2SpecServer</mainClass>
           <skip>${skipTests}</skip>
+          <junitPackage>org.eclipse.jetty.h2spec</junitPackage>
+          <excludeSpecs>
+            <excludeSpec>3.5 - Sends invalid connection preface</excludeSpec>
```

Comment:
Is this right?
Are we actually excluding this specific test?
If so, perhaps we should include a `<!-- comment explaining why -->`?

Reply:
@lorban do you remember why we need to exclude this test?

Reply:
If you have time otherwise we only need a comment.